scripts/ping_slack_about_package_release.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+if [[ ${TRAVIS_TAG} =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+    curl -X POST -H "Content-type: application/json" \
+	 --data "{\"text\":\"💫 New *Rasa SDK* version ${TRAVIS_TAG} has been released! Changelog: https://github.com/RasaHQ/rasa-sdk/blob/${TRAVIS_TAG}/CHANGELOG.rst\"}" \
```

Comment:
let's use a different emoji e.g.
```suggestion
	 --data "{\"text\":\"😻 New *Rasa SDK* version ${TRAVIS_TAG} has been released! Changelog: https://github.com/RasaHQ/rasa-sdk/blob/${TRAVIS_TAG}/CHANGELOG.rst\"}" \
```

Reply:
Haha it is different, the other use ⚡️ and 💥 . I like the cat though 😄